### PR TITLE
Add explanation about boolean configurations in env/fs

### PR DESCRIPTION
**Pull request by aviramha**

### CODE REVIEW

File: mirrord/config/src/feature/env.rs [hunk 1]
```diff
@@ -12,6 +12,11 @@ use crate::{
 
 /// Allows the user to set or override the local process' environment variables with the ones
 /// from the remote pod.
+/// Can be set to one of the options:
```
Comment (meowjesty) [inline mirrord/config/src/feature/env.rs:15]:
```suggestion
///
/// Can be set to one of the options:
```

Extra space for the rendered markdown.

**Review by t4lz — APPROVED** [review]
please also run medschool

File: mirrord/config/src/feature/fs/advanced.rs [hunk 2]
```diff
@@ -17,9 +17,9 @@ use crate::{
 
 /// Allows the user to specify the default behavior for file operations:
 ///
-/// 1. `"read"` - Read from the remote file system (default)
+/// 1. `"read"`/`true` - Read from the remote file system (default)
```
Comment (meowjesty) [inline mirrord/config/src/feature/fs/advanced.rs:20]:
```suggestion
/// 1. `"read"` or `true` - Read from the remote file system (default)
```

I think it's a bit more readable like this.

File: mirrord/config/src/feature/fs/advanced.rs [hunk 3]
```diff
@@ -17,9 +17,9 @@ use crate::{
...
 /// 2. `"write"` - Read/Write from the remote file system.
-/// 3. `"local"` - Read from the local file system.
+/// 3. `"local"`/`false` - Read from the local file system.
```
Comment (meowjesty) [inline mirrord/config/src/feature/fs/advanced.rs:22]:
```suggestion
/// 3. `"local"` or `false` - Read from the local file system.
```

**Review by meowjesty — APPROVED** [review]
Docs nits.

Can you also run medschool to update the `configuration.md` please.

**Pull request merged**